app/src/main/kotlin/com/wire/android/ui/home/conversationslist/common/ConversationItemFactory.kt

```diff
@@ -53,6 +54,9 @@ import com.wire.kalium.logic.data.user.UserId
 fun ConversationItemFactory(
     conversation: ConversationItem,
     searchQuery: String,
+    isSelectableItem: Boolean = false,
+    belongsToGroup: Boolean = false,
```

Comment:
not cleart to me what belongsToGrouop means here

Reply:
this variable is meant to represent if the item generated should be marked as selected or not. But since we are only allowing one conversation items to be selected within a radio group, I guess I can rename it to `isChecked`. If we decide in the future that several conversation items could be selected, then we can rename to `isSelected`